app/views/layouts/application.html.erb

```diff
@@ -11,6 +11,8 @@
           description
     %>
 
+    <link rel="canonical" href="http://budgetingkid.com/">
```

Comment:
Moving further we will have more public pages. Every page has its all uniq path. So canonical for them should be also uniq. e.g. a policy page is http://budgetingkid.com/policy and canonical is http://budgetingkid.com/policy for it as well.